README.md

```diff
@@ -22,7 +22,7 @@ use:
 <dependency>
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>1.0.6</version>
+  <version>1.0.7</version>
```

Comment:
Should this version be updated to the latest stable release, which I assume is 1.0.8
```suggestion
  <version>1.0.8</version>
```